src/app/post/[post_id]/mobile/components/ExperimentPostMobileContentContainer/ExperimentPostMobileContentContainer.tsx

```
@@ -0,0 +1,117 @@
+  postId: string;
+}) => {
+  const { open, close } = useOverlay();
+  const [isCopyToastOpen, setIsCopyToastOpen] = useState(false);
+
+  const postDetailData = experimentDetailResponse.data;
+
+  if (experimentDetailResponse.isLoading) {
+    return (
+      <div className={emptyView}>
+        <Spinner />;
+      </div>
```

Biome (1.9.4)

[error] There is a suspicious semicolon in the JSX element.

This is usually the result of a typo or some refactor gone wrong.
Remove the semicolon, or move it inside a JSX element.

(lint/suspicious/noSuspiciousSemicolonInJsx)